src/pages/Top.tsx

```diff
@@ -0,0 +1,15 @@
+import { Register } from "../components/Register";
+import { Login } from "../components/Login";
+import main from "../assets/images/tv-reminder-mainvisual.png";
+
+export const Top = () => {
+  return (
+    <div className="top_container">
+      <div className="top_img_main">
```

Comment:
ここだけアンスコになってますが何か制約ありますかね？

```suggestion
    <div className="top-container">
      <div className="top-img-main">
```